### ISSUE TITLE
Victory of the Calls of the Prophets and Scholars

### ISSUE THREAD

**Issue by aghamidi** [aghamidi/Project#203]
http://www.youtube.com/watch?v=IaayjnY0N24

What is the topic of the lecture (series)? (3-5 sentences. Don't copy the title.)
Why is this topic important according to the lecturer (3-5 sentences)?

**Comment by ajghamidi** [comment]
This that

**Comment by ajghamidi** [comment]
Test

**Comment by ajghamidi** [comment]
Test 2

**Comment by fawaz-adami** [comment]
Having the correct foundations for 'imaan is of paramount importance. By having it, the 'imaan of a person will be based on that which is firm, and everything that stems from it will be correct. The best way to have the right 'imaan is to have the correct, and beneficial, knowledge along with the required implementation of that knowledge. Shaikh Mustapha al-Mazjoub explained what exactly is the correct knowledge, and methodology, to obtaining the right foundations for our 'imaan, and thus how to implement it correctly.

**Comment by fawaz-adami** [comment]
The correct foundations of 'imaan lead to performing acceptable deeds. The best way to have the right 'imaan is to have the correct, and beneficial, knowledge along with the required implementation of that knowledge. In this lecture, Shaikh Mustapha al-Mazjoub explains what correct 'imaan is, how to obtain it and the way to implement knowledge correctly so that 'imaan and the subsequent deeds that are based off it are both correct.

**Comment by aghamidi** [comment]
@ummhurairah , below are the instructions on proofreading. Please copy and save them on your computer because it would be quite messy for me to add them to each issue.
Please save the attached excel document with the transliteration for the same reason.

Proofreading instructions:
Please correct all typos, grammatical errors and awkward phrasings that you notice.

Please make sure that the summary provides a) an overview of the content of the lecture (series) and b) a description of why the specific topic is important for Muslims to study.

Please make sure that the summary consists of between 450-550 characters in total.

Please make sure that ALL Arabic words are transliterated according to the scheme in the attached file. (The first sheet describes the transliteration of individual letters while the second sheet contains a list of words that we've already transliterated which you can reference). This is to ensure consistency across the website.

[Dictionary.xlsx](https://github.com/aghamidi/Project/files/1580153/Dictionary.xlsx)

**Comment by ummhurairah** [comment]
The correct foundations of ʾīmān lead to performing acceptable deeds. The best way to have the right ʾīmān is to have the correct, and beneficial, knowledge along with the required implementation of that knowledge. In this lecture, Shaikh Mustapha al-Mazjoub explains what correct ʾīmān is, how to obtain it, and the way to implement knowledge correctly so that ʾīmān and the subsequent deeds that are based on it are correct.

**Comment by aghamidi** [comment]
Well done. If this setup works for you, we can keep assigning texts to proofread but in much larger quantities at once.

One thing that I forgot to point out is that we want the first letter of all Arabic words capitalized, i.e. ʾĪmān instead of ʾīmān.

The transliteration of "Shaikh Mustapha al-Mazjoub" is incorrect though and should be "Shaykh Mustapha Al Majzoub". Please make sure you transliterate in a consistent way going forward by referring to the spreadsheet. I know it's annoying at first but after a while it should come naturally and will help us maintain a consistent language in shaa Allah.

I will make the correction here and on other summaries you proofread in shaa Allah.

**Comment by aghamidi** [comment]
One final thing, the word "Allah" and the names of our four lecturers (Anwar Al-Awlaki, Ahmad Musa Jibril, Ali Al-Tamimi and Mustapha Al-Majzoub) are the only words that we won't transliterate in shaa Allah and that has to do with search engine optimization.

**Comment by ummhurairah** [comment]
I will keep these points in mind. Please inform me if I miss things so I can keep those in mind.

Yes, go ahead and assign more. Do you think you could re-assign the audio summaries to others in that case?

**Comment by aghamidi** [comment]
Yes, I will in shaa Allah just assign you proofreading tasks going forward. The texts that you will be asked to proofread can vary from summaries to "About-pages" to lecture notes and so forth. I hope that is fine.

**Comment by aghamidi** [comment]
One more thing that I noticed: the character limitations vary a bit too much and two are a bit below the target of 450-550 (Victory of the Calls of the Prophets and Scholars=426 and Anger Management=437). Let's revise the target to 500-550 going forward. Would you mind fixing those two summaries?

I know that in some cases, the written material that you've been provided is not sufficient. In those rare cases, could you please try to listen quickly to bits of the lecture to supplement the other material?

Barak Allah feekum for your patience.

**Comment by fawaz-adami** [comment]
The correct foundations of ʾĪmān lead to performing acceptable deeds. Many today have based their ʾĪmān on knowledge that is neither correct nor useful. The best way to have the right ʾĪmān is to have the correct, and beneficial, knowledge along with the required implementation of that knowledge. In this lecture, Shaykh Mustapha Al-Majzoub explains what correct ʾĪmān is, how to obtain it, and the way to implement knowledge correctly so that ʾĪmān and the subsequent deeds that are based on it are correct.

**Comment by fawaz-adami** [comment]
People understand victory in different ways, each according to what they are influenced by. Victory means one thing to the disbeliever, while it means something else in the context of 'Islam and Muslims. However, many Muslims prioritize material victory, when the victory of the mission of the Prophets, and the work of the scholars, is in the victory of the ideology. Shaikh Mustapha al-Majzoub explains between the two definitions and explains the victory of the Prophets and scholars in an 'Islamic context.
